app/views/messages/index.html.erb

```diff
@@ -8,7 +8,12 @@
   <h2>Contact History</h2>
   <% if @contact_history.present? %>
     <ul class="contact-history">
+      <% last_date = nil %>
       <% @contact_history.each do |contact_record| %>
+        <% if contact_record.datetime.to_date != last_date %>
+          <li class="contact-record__day-heading"><%= if contact_record.datetime.to_date == Date.today then "Today" end %> <%= contact_record.datetime.to_date.strftime("%m/%d/%Y") %></li>
```

Comment:
Putting "Today" into a translation string would help ensure we don't miss it later